chart/values.yaml

```diff
@@ -106,7 +106,7 @@ clustersysteminfo:
     OWSEC_USERNAME: tip@ucentral.com
     OWSEC_PASSWORD: openwifi
 
-  activeDeadlineSeconds: 1200
+  activeDeadlineSeconds: 2400
   backoffLimit: 5
```

Comment:
Probably we should also increase this value + we would also need to increase wait time in deploy script (https://github.com/Telecominfraproject/wlan-testing/blob/master/helm/ucentral/deploy.sh#L95)

Reply:
Think 5 times 20 minutes should be sufficient.